ush/fv3gfs_downstream_nems.sh

```diff
@@ -1,294 +1,179 @@
-export CNVGRIB=${CNVGRIB:-${grib_util_ROOT}/bin/cnvgrib}
-export COPYGB2=${COPYGB2:-${grib_util_ROOT}/bin/copygb}
+# Programs used
 export WGRIB2=${WGRIB2:-${wgrib2_ROOT}/bin/wgrib2}
-export GRBINDEX=${GRBINDEX:-${wgrib2_ROOT}/bin/grbindex}
-export RUN=${RUN:-"gfs"}
-export cycn=$(echo $CDATE |cut -c 9-10)
-export TCYC=${TCYC:-".t${cycn}z."}
-export PREFIX=${PREFIX:-${RUN}${TCYC}}
-export PGB1F=${PGB1F:-"NO"}
-export FHOUT_PGB=${FHOUT_PGB:-3}
-export PGBS=${PGBS:-"NO"} #YES-- generate 1.00 and 0.50 deg pgb data
-export MODICEC=${MODICEC:-$USHgfs/mod_icec.sh}
-
-#--wgrib2 regrid parameters
-export option1=' -set_grib_type same -new_grid_winds earth '
-export option21=' -new_grid_interpolation bilinear  -if '
-export option22=":(CRAIN|CICEP|CFRZR|CSNOW|ICSEV):"
-export option23=' -new_grid_interpolation neighbor -fi '
-export option24=' -set_bitmap 1 -set_grib_max_bits 16 -if '
-export option25=":(APCP|ACPCP|PRATE|CPRAT):"
-export option26=' -set_grib_max_bits 25 -fi -if '
-export option27=":(APCP|ACPCP|PRATE|CPRAT|DZDT):"
-export option28=' -new_grid_interpolation budget -fi '
-export grid0p25="latlon 0:1440:0.25 90:721:-0.25"
-export grid0p5="latlon 0:720:0.5 90:361:-0.5"
-export grid1p0="latlon 0:360:1.0 90:181:-1.0"
-export grid2p5="latlon 0:144:2.5 90:73:-2.5"
-
-unset paramlist paramlistb
-if [ $FH -eq -1 ] ; then
-  #export paramlist=/global/save/Hui-Ya.Chuang/gfs_trunk/sib/fix/global_1x1_paramlist_g2.anl
-  export paramlist=${paramlist:-$PARMpost/global_1x1_paramlist_g2.anl}
-  export paramlistb=${paramlistb:-$PARMpost/global_master-catchup_parmlist_g2}
-  export fhr3=anl
-  export PGBS=YES
-elif [ $FH -eq 0 ] ; then
-  export paramlist=${paramlist:-$PARMpost/global_1x1_paramlist_g2.f000}
-  export paramlistb=${paramlistb:-$PARMpost/global_master-catchup_parmlist_g2}
-  export fhr3=000
-  export PGBS=YES
+CNVGRIB=${CNVGRIB:-${grib_util_ROOT}/bin/cnvgrib}
+GRBINDEX=${GRBINDEX:-${wgrib2_ROOT}/bin/grbindex}
+
+# Scripts used
+GFSDWNSH=${GFSDWNSH:-"${HOMEgfs}/ush/fv3gfs_dwn_nems.sh"}
+
+# variables used here and in $GFSDWNSH
+PGBOUT2=${PGBOUT2:-"master.grib2"}  # grib2 file from UPP
+FH=$(( ${FH:-0} ))  # Forecast hour to process
+FHOUT_PGB=${FHOUT_PGB:-3}  # Output frequency of GFS PGB file at 1-degree and 0.5 degree
+npe_dwn=${npe_dwn:-24}
+downset=${downset:-1}
+PREFIX=${PREFIX:-"${RUN:-gfs}.t${cyc}z."}
+PGBS=${PGBS:-"NO"}  # YES - generate 1 and 1/2-degree grib2 data
+PGB1F=${PGB1F:-"NO"}  # YES - generate 1-degree grib1 data
+
+# Files used
+if (( FH == -1 )); then
+  fhr3=anl
+  PGBS="YES"
+  paramlista=${paramlist:-"${HOMEgfs}/parm/post/global_1x1_paramlist_g2.anl"}
+elif (( FH == 0 )); then
+  fhr3=000
+  PGBS="YES"
+  paramlista=${paramlist:-"${HOMEgfs}/parm/post/global_1x1_paramlist_g2.f000"}
 else
-  export paramlist=${paramlist:-$PARMpost/global_1x1_paramlist_g2}
-  export paramlistb=${paramlistb:-$PARMpost/global_master-catchup_parmlist_g2}
-  export fhr3=$(printf "%03d" ${FH})
+  fhr3=$(printf "%03d" "${FH}")
```

Comment:
Is `$FH` similar to the `$fhr` variable used in the post ex-scripts? If so, should the variables be aligned? See PR #1859.

Reply:
`FH` is coming from above this level.  I am working from this level down, so if `FH` above gets updated to something, I am happy to update here.

Reply:
Roger that! Thanks for explaining. :)